synapse/handlers/_base.py

```diff
@@ -53,9 +53,15 @@ def __init__(self, hs):
         self.event_builder_factory = hs.get_event_builder_factory()
 
     @defer.inlineCallbacks
-    def _filter_events_for_clients(self, user_tuples, events, event_id_to_state):
+    def filter_events_for_clients(self, user_tuples, events, event_id_to_state):
         """ Returns dict of user_id -> list of events that user is allowed to
         see.
+
+        :param (str, bool) user_tuples: (user id, is_peeking) for each
```

Comment:
Param style is wrong here; should be:

Args:
    argname (type): Description

(and below)